lib/general.sh

```diff
@@ -204,8 +204,16 @@ fetch_from_repo()
 	local ref=$3
 	local ref_subdir=$4
 
+	# The 'offline' variable must always be set to 'true' or 'false'
+	if [ "$OFFLINE_WORK" == "yes" ]; then
```

Comment:
OFFLINE_WORK
Also add here https://github.com/armbian/build/blob/master/lib/build-all-ng.sh#L60 and 
https://github.com/armbian/documentation/blob/master/docs/Developer-Guide_Build-Options.md

Reply:
OK!

Reply:
[.ignore_changes](https://github.com/armbian/documentation/blame/master/docs/Developer-Guide_Build-Options.md#L79)
This strange entity made me think.
It is only [present here](https://github.com/armbian/build/blob/master/compile.sh#L44)
But this code in my build system never works, because the [systemd-detect-virt](https://github.com/armbian/build/blob/master/compile.sh#L77) command always returns "kvm"
For me, the lack of this functionality is a great boon, updating the build system manually is what I most want.
What should I do in this script?

Reply:
do you run your build on cloud?

Reply:
If the build system is in the cloud, there are no problems with the Internet, and there are no problems with the build speed. In this case, I wouldn't be doing this job.
I have openSUSE working on my host and use qemu-kvm and virt-manager as a VM.